lib/compilers/WSL-VC.js

```diff
@@ -24,20 +24,22 @@
 
 
 // AP: WSL-CL.js is a WSL-specific copy of CL.js (renamed to Wine-CL.js). 
+// Nicole: these were renamed later to X-VC.js, since the compiler is named VC
```

Comment:
I'd be happy enough removing these comments; the heritage of the file is less important than what it's doing now. Ideally we would extract any commonality left between the files (if any exists). But I don't propose we do that in this change.